### ISSUE TITLE
Design for TokenScript for attestation tokens

### ISSUE THREAD

**Issue by AW-STJ** [TokenScript/attestation#205]

**Comment by oleggrib** [comment]
List of the properties to describe in TokenScript:
- TokenName
- TokenDescription
- TokenWebsiteURL
- Token schema (property->type)
- attestationOrigin
- AttestorPublicKey
- tokenOrigin
- magicLinkParamsSchema (token, secret, pok ,id, type)
- localStorageDBName
- localStorageTokenItemName
- localStorageEthKeyItemName
- tokenUniqueIdParam
- IssuerPublicKey
- WhiteListDomainNames

@nicktaras , do you need some additional properties in the TokenScript to describe specific token?

**Comment by oleggrib** [comment]
I can see config data in the Readme.md in the https://github.com/TokenScript/token-negotiator

those data must be syncronized with backend validator, other services who use this token etc.

Also webster have to know some tech info about this Token and can make mistake somewhere.

In case if we finish this task then we can pass just tokenName to the Negotiator, other data can be fetched from remote XML file.

Lets do next:
- define fields list 
- create XML file with embedded token data
- copy XML parser code from https://testshop.antopolbus.rv.ua/ts3/passive-negotiation.html, its vanilla JS, I have improved version in TS as angular service

@nicktaras @micwallace @foxgem @AW-STJ

**Comment by micwallace** [comment]
Yep 100% agree Oleh. As soon as we get this into TokenScript we can take advantage of the TokenScript signing security.

**Comment by oleggrib** [comment]
@micwallace , @nicktaras do you work on this issue?
if no then lets make simple JSON file (as forst draft) in specific location and read it from there. ok? I can do the PR.

**Comment by oleggrib** [comment]
@micwallace , @nicktaras will we still duplicate config? out backend should validate it too, so we need aligned config. With more duplication more chanses make mistake.

**Comment by AW-STJ** [comment]
I will escalate this with @weiwu-zhang. We had discussed this recently as well.

If we are hitting this issue again, and is related to attestation.id as well, then its high time to decide on the config before moving forward
